### DISCUSSION TITLE
"Package not scheduled" although workflow run is within schedule

### DISCUSSION THREAD

**Discussion by beatngu13** [renovatebot/renovate#31944] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub (Action: 40.3.1, Renovate: 38.121.0)

### Please tell us more about your question or problem

Our self-hosted renovate workflow has the following `schedule`:

```yaml
cron: '*/60 06-19 * * 1-5'
```

That is, it runs every hour between 6 am and 7 pm from Monday to Friday. The schedule of the shared Renovate config is:

```json
"schedule": [
    "before 12am on Monday"
],
```

The problem is that the branches/PRs are not created due to "Package not scheduled" (see logs).

Did we mess up something in our config?

### Logs (if relevant)

<details><summary>Logs</summary>

```
[...]
DEBUG: Checking schedule(schedule=before 12am on Monday, tz=null, now=2024-10-14T09:52:56.886Z) (repository=l-bank/integration-sap-gp, branch=l-bank-renovate/apache-maven-plugins)
DEBUG: Checking 1 schedule(s) (repository=l-bank/integration-sap-gp, branch=l-bank-renovate/apache-maven-plugins)
DEBUG: Checking schedule "before 12am on Monday" (repository=l-bank/integration-sap-gp, branch=l-bank-renovate/apache-maven-plugins)
       "parsedSchedule": {"schedules": [{"t_b": [0], "d": [2]}], "exceptions": [], "error": -1}
DEBUG: Package not scheduled (repository=l-bank/integration-sap-gp, branch=l-bank-renovate/apache-maven-plugins)
DEBUG: Skipping branch creation as not within schedule (repository=l-bank/integration-sap-gp, branch=l-bank-renovate/apache-maven-plugins)
[...]
DEBUG: Renovate repository PR statistics (repository=l-bank/integration-sap-gp)
       "stats": {"total": 5, "open": 0, "closed": 3, "merged": 2}
DEBUG: Repository result: done, status: activated, enabled: true, onboarded: true (repository=l-bank/integration-sap-gp)
```

</details>

**Comment by rarkins** [comment, marked as answer]
12am is the start of the day, there is no such thing as "before 12am" in a day.

Recommendation: switch to cron-style schedule in Renovate config too
